pkg/connectors/datacatalog/clients/datacatalog_openapi.go

```diff
@@ -45,7 +45,7 @@ func NewOpenAPIDataCatalog(name, connectionURL string) DataCatalog {
 			},
 		},
 		OperationServers: map[string]openapiclient.ServerConfigurations{},
-		HTTPClient:       utils.GetHTTPClient(&log),
+		HTTPClient:       utils.GetHTTPClient(&log).StandardClient(),
```

Comment:
I see that GetHTTPClient is not used without StandardClient(), so why not to change the utils.GetHTTPClient to return the final object?

Reply:
In opametadata-connector code which uses this function not all calls will use StandardClient() (for example in Vault client). The above code needs to use StandardClient() because it uses autogenrated code.

Reply:
It is a problem. Whoever is looking at the code in fybrik repo has no idea that the same code is also used elsewhere in a different flavor. A cleaner way would be to create a separate package (with its own go.mod). It does not make sense for openmedata-connector to import the entire fybrik just to use a package or two.

Reply:
@shlomitk1 OK, we can also create a separate package for vault and tls pkgs which the modules can also use. Discussing this with @roytman we might want to do that after the packages will be stabilized (there are functions in vault packages that need to be moved from openmetadata repo)